client/packages/system/src/IndicatorsDemographics/api/hooks/document/useDemographicProjections.ts

```diff
@@ -1,14 +1,33 @@
-import { useUrlQueryParams, useQuery } from '@openmsupply-client/common';
+import { useQuery } from '@openmsupply-client/common';
 import { useDemographicsApi } from '../utils/useDemographicApi';
+import { DemographicProjectionFragment } from '../../operations.generated';
 
-export const useDemographicProjections = () => {
-  const { queryParams } = useUrlQueryParams({
-    filters: [{ key: 'name' }, { key: 'basePopulation' }, { key: 'id' }],
-  });
+export const useDemographicProjections = (baseYear: number) => {
   const api = useDemographicsApi();
-  const filterBy = queryParams.filterBy;
-  const params = { ...queryParams, filterBy };
-  return useQuery(api.keys.paramProjectionList(params), () =>
-    api.getProjections.list(params)
-  );
+  const params = {
```

Comment:
we aren't using urlQueryParams so I removed that and tidied this to the essentials